index.html

```diff
@@ -0,0 +1,179 @@
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-desktop"></i>
+                    <h4>Online Presentation</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div>
+                    <i class="fas fa-certificate"></i>
+                    <h4>Certificate</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-stamp"></i>
+                    <h4>Job Guarantee</h4>
+                </div>
+
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-chalkboard-teacher"></i>
+                    <h4>Strong Team</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+        </ul>
+        <div class="program-paragraph">
+            <a href="#">Join CC Global Summit 2015</a>
+        </div>
+        <div class="program-paragraph-desktop">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+    <div class="program-speakers-title">
+        <h2>Featured Speakers</h2>
+        <hr>
+    </div>
+
+    <section class="program-speakers">
+
+ 
+       
+
+    </section>
+    <section class="program-speakers">
+        <div class="speakers-more">
+            <button id="loadMore"> More <i class="fas fa-arrow-down"></i></button>
+        </div>
+    </section>
+   
+    <section class="partners">
+        <div class="program-speakers-title">
+            <h2>Partner</h2>
+            <hr>
+        </div>
+        <div class="partners-icons">
+            <ul>
+                <li class="partners-icon">
+                    <img src="img/example-2.png" alt="sponsor">
+                </li>
+                <li class="partners-icon">
+                    <img src="img/example-2.png" alt="sponsor">
+                </li>
+                <li class="partners-icon">
+                    <img src="img/example-2.png" alt="sponsor">
+                </li>
+
+            </ul>
+        </div>
+    </section>
+    <footer>
+        <div class="footer-logo">
+            <i class="fas fa-code"></i>
+        </div>
+        <div class="footer-text">
+            <p>2021 Web Developer Course All Rights Reserverd</p>
+            <p>Best WebDeveloper Course Template</p>
+
+
+        </div>
+
+    </footer>
```

Comment:
- [ ] Footer is not present on your page in the mobile size view. It should contain a logo aligned to the left and text aligned to the right and white background.

![Screenshot from 2021-10-20 14-03-35](https://user-images.githubusercontent.com/40503704/138098217-fb80181c-4717-4b87-beb1-f81988517d86.png)